src/components/filterControl.tsx

```diff
@@ -23,7 +23,11 @@ function FilterControl(){
         setState({ ...state, filterFrequency: newValue });
     }
 
-    const { filterFrequency } = state;
+    const setFilterQAndState = (newValue: number) => {
```

Comment:
Not necessary for this PR, but we should rename these methods. 
At one point we were both setting Context state AND actually triggering the change in these control components. Now that we only set the state and allow the hooks wrapper to trigger the change, so we can rename these methods "setFilterState" or something like that

Reply:
+1, good suggestion